php/classes/read/random.php

```diff
@@ -3,48 +3,17 @@
 require_once './classes/HTTPException.php';
 
 function make_seed() {
-    list($usec, $sec) = explode(' ', microtime());
+    [$usec, $sec] = explode(' ', microtime());
     return intval($sec + $usec * 1000000);
 }
 
-// can run with a maximum amount of random entries
-// (if collection is smaller it's not guaranteed)
-// (is optional, else it will be all the results)
-function random($params, $class) {
```

Comment:
I wanted it in the random part not the JSON database part for a reason, JSONDatabase is an implementation of the results